cfgov/unprocessed/apps/paying-for-college/js/disclosures/index.js

```diff
@@ -83,6 +84,9 @@ const app = {
             // Update expenses model bases on region and salary
             const region = schoolValues.BLSAverage.substr(0, 2);
             $('#bls-region-select').val(region).change();
+            $step2.removeClass('a-btn__disabled');
+            $step2.text('Continue to Step 2');
+            $step2.elements[0].disabled = null;
```

Comment:
I think you want `step2.elements[0].removeAttribute('disabled')`

Reply:
Cool. I am happy to do it this way.